docs/conf.py

```diff
@@ -108,23 +101,14 @@ def render_hubs():
 
         # For each hub in cluster, grab its metadata and add it to the list
         for hub in cluster["hubs"]:
-            config = hub["config"]
-            # Config is sometimes nested
-            if "basehub" in config:
-                hub_config = config["basehub"]["jupyterhub"]
-            else:
-                hub_config = config["jupyterhub"]
             # Domain can be a list
             if isinstance(hub["domain"], list):
                 hub["domain"] = hub["domain"][0]
 
             hub_list.append(
                 {
-                    "name": hub_config["custom"]["homepage"]["templateVars"]["org"][
-                        "name"
-                    ],
-                    "domain": f"[{hub['domain']}](https://{hub['domain']})",
                     "id": hub["name"],
+                    "domain": f"[{hub['domain']}](https://{hub['domain']})",
```

Comment:
This will be even further broken after we also move the support config out of the `cluster.yaml` file since it also scrapes the grafana URL. I'll fix it after that change has been made and the config is stable.